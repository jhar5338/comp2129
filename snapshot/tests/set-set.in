set a
get a
set a 1
get a
bye
