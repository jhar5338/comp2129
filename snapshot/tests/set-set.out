> ok

> []

> ok

> [1]

> bye
